app/Http/Controllers/AuthController.php

```diff
@@ -59,7 +67,7 @@ public function register(Request $request){
 
     public function logout(Request $request)
     {
-        auth('sanctum')->user()->tokens()->delete();
+        Auth::user()->tokens()->delete();
```

Comment:
Do not delete all tokens. Delete only the current one. The one that user is using to be authenticated rn